docs/getting-started.md

```diff
@@ -243,3 +243,22 @@ TBD
 Thanos also has a tutorial on deploying it to Kubernetes. We have a full page describing a standard deployment here.
 
 We also have example Grafana dashboards [here](/examples/grafana/monitoring.md) and some [alerts](/examples/alerts/alerts.md) to get you started.
+
+## Single host port layout
```

Comment:
Can we change this to  maybe something like:

```
We don't recommend running Thanos on a single node on production. Thanos is designed and built to run as a distributed system. Vanilla Prometheus might be totally enough for small setups.

However, in case you want to play and run Thanos components on a single node, we recommend following the port layout: 

< TABLE> 

You can see example one-node setup [here](./scripts/quickstart.sh)
```
Plus, we might want to adjust script for those ports (:

Reply:
Not sure if the statement isn't bit too harsh because I can imagine using Thanos just as a global view solution when just sidecar and query is running on along with the Prometheus on the same host and having two of those hosts providing also HA for query but that is probably a corner case. 

I'll update it to your suggestions  and definitely also the script, I completely forgot about it thanks for reminding!

Reply:
> Not sure if the statement isn't bit too harsh because I can imagine using Thanos just as a global view solution when just sidecar and query is running on along with the Prometheus on the same host and having two of those hosts providing also HA for query but that is probably a corner case.

Well for HA you probably want more than one Querier as well and not on the same host. Anyway feel free to adjust those suggestions to be less harsh, I definitely don't mean to be harsh (:

Reply:
I meant two hosts and every one one of them running Prometheus, sidecar and query.
Harsh was a probably bit too harsh word :D but anyway I like the testing usage formulation so I'll stick with yours version.